src/Illuminate/Auth/TokenGuard.php

```diff
@@ -18,11 +18,11 @@ class TokenGuard implements Guard
     protected $request;
 
     /**
-     * The name of the field on the request containing the API token.
+     * The name of the query string item from the request containing the API token.
      *
      * @var string
      */
-    protected $inputKey;
+    protected $queryKey;
```

Comment:
Thanks for letting me know that information. I added a commit reverting back to the old variable name. So now all thats changed is the request->query access and the variable's comment.